spec/lib/mvi/service_spec.rb

```diff
@@ -321,4 +321,30 @@
       end
     end
   end
+
+  describe '.find_profile monitoring' do
+    context 'with a successful request' do
+      it 'should increment find_profile total' do
+        allow(user).to receive(:mhv_icn).and_return('1008714701V416111^NI^200M^USVHA^P')
+
+        allow(StatsD).to receive(:increment)
+        VCR.use_cassette('mvi/find_candidate/valid') do
+          subject.find_profile(user)
+        end
+        expect(StatsD).to have_received(:increment).with('api.mvi.find_profile.total')
```

Comment:
Do we need both `allow(StatsD).to receive(:increment)` and `expect(StatsD).to have_received(:increment)`? I usually only use `allow to receive` if I don't have an `expect to receive` statement.

Reply:
`:increment` is getting called multiple times. Once by mvi and then once by a service called `StatsdPlugin`. Because of this, I needed it to be stubbed out via allow and then check to see if it received the correct call afterwards (via `have_received`). Using `expect().to receive...` gave me a mismatch.

Reply:
Perfect! I figured you probably had a good reason :)